dist/spec.yaml

```diff
@@ -9203,7 +9206,7 @@ definitions:
       x5t:
         readOnly: true
         type: string
-      'x5t#S256':
+      x5t#S256:
```

Comment:
is `#` symbol intended to be a part of property name? (it requires extra handling during SDK generation)

Reply:
Yes. It is